tasks/main.yml

```diff
@@ -70,7 +70,7 @@
         opts: bind
         state: mounted
         fstype: none
-      when: nix_bind_mount_directory is changed
+      when: nix_bind_mount_directory is changed  # noqa: no-handler
```

Comment:
I guess b0d9b4954a9ad1c173479153ad897fb10ca8b9b3 should be squashed into 3247e98e933c6ab44178fdbd7111c254ac138258, right?